src/css/_modebar.scss

```diff
@@ -44,19 +39,21 @@
         top: 2px;
     }
 
-    path {
-        fill: rgba(0,31,95,0.3);
-    }
+    &.modebar-btn--logo {
 
-    &.active path, &:hover path {
-        fill: rgba(0,22,72,0.5);
     }
+}
 
-    &.modebar-btn--logo {
-        padding: 3px 1px;
+.modebar.vertical {
+    .modebar-group {
+        display: block;
+        float: none;
+        margin-left: 0px;
+        margin-top: 8px;
```

Comment:
![peek 2018-10-04 13-41](https://user-images.githubusercontent.com/6675409/46492342-3f62b980-c7db-11e8-97c1-960ecc599657.gif)

Looks like the vertical orientation has more top margin than the what the horizontal orientation has to its right.

Is this on purpose? This is just something I noticed, I don't have a strong opinion on what this _should_ look like.

Reply:
Nice catch 👀 ! No, it's not on purpose. I'd rather have the logo at the top. Fixing this right now!